Gemfile

```diff
@@ -5,9 +5,10 @@ ruby '2.2.4'
 gem 'rails', '4.2.1'
 gem 'puma'
 
-gem 'pg'
+gem 'pg', '~> 0.18'
```

Comment:
Can you comment on the motivation for the version upgrades of pg, paperclip, apartment? Not sure whether this is deployed to staging, but given that we're having trouble on staging, it seems like these upgrades should be in their own PR, rather than rolled into a PR related to whitespace.